go.mod

```diff
@@ -72,3 +72,5 @@ require (
 	k8s.io/api v0.22.1
 	k8s.io/apimachinery v0.22.1
 )
+
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1
```

Comment:
Why `replace`?  That freezes it in time to v1.11.1 even if newer versions are available.

Reply:
ah ok, I was looking to remove older versions from go.sum . Is there a keyword to remove older versions and yet not block from updates?

Reply:
I don't think we need to bother about `go.sum`. If we add `github.com/prometheus/client_golang v1.11.1` it will be the lower boundary but allow newer versions to come in.

Reply:
ack, updating ..